### PR TITLE
TeamViewer.132 version 1.0

### PR DESCRIPTION
**Pull request by RanierBrolessi**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90360)

### CODE REVIEW

**Review by russellbanks — CHANGES_REQUESTED** [review]
TeamViewer is already in winget-pkgs under `TeamViewer.TeamViewer`

File: manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TeamViewer.132
```
Comment (stephengillie) [inline manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml:4]:
```suggestion
PackageIdentifier: TeamViewer.TeamViewer
```

File: manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TeamViewer.132
+PackageVersion: 1.0
```
Comment (stephengillie) [inline manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml:5]:
```suggestion
PackageVersion: 13.2.36224
```

File: manifests/t/TeamViewer/132/1.0/TeamViewer.132.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: TeamViewer.132
```
Comment (stephengillie) [inline manifests/t/TeamViewer/132/1.0/TeamViewer.132.yaml:4]:
```suggestion
PackageIdentifier: TeamViewer.TeamViewer
```

File: manifests/t/TeamViewer/132/1.0/TeamViewer.132.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: TeamViewer.132
+PackageVersion: 1.0
```
Comment (stephengillie) [inline manifests/t/TeamViewer/132/1.0/TeamViewer.132.yaml:5]:
```suggestion
PackageVersion: 13.2.36224
```

File: manifests/t/TeamViewer/132/1.0/TeamViewer.132.locale.pt-BR.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: TeamViewer.132
+PackageVersion: 1.0
```
Comment (stephengillie) [inline manifests/t/TeamViewer/132/1.0/TeamViewer.132.locale.pt-BR.yaml:5]:
```suggestion
PackageVersion: 13.2.36224
```

File: manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TeamViewer.132
+PackageVersion: 1.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://dti.criciuma.sc.gov.br/arquivos/TeamViewer13_COMPLETO_DTI.exe
```
Comment (stephengillie) [inline manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml:9]:
```suggestion
  InstallerUrl: https://download.teamviewer.com/download/version_13x/TeamViewer_Setup.exe?_ga=2.18429113.1619990487.1670013023-1195273765.1670013023

```

File: manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TeamViewer.132
+PackageVersion: 1.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://dti.criciuma.sc.gov.br/arquivos/TeamViewer13_COMPLETO_DTI.exe
+  InstallerSha256: 7BD793996D5F6F775830DECC9F4FD08A41DAC4F8B7BE77AE800DC38B38DFE6CF
```
Comment (stephengillie) [inline manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml:10]:
```suggestion
  InstallerSha256: 0E40E0D4C3966EF61C84985C5819A10661F1DD972B74FFA00F4B8422A4E5BBC6

```

File: manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml (hunk 6 above)
Comment (stephengillie) [reply manifests/t/TeamViewer/132/1.0/TeamViewer.132.installer.yaml:9]: Installers must come from the publisher's website.

**Pull request closed without merging**